wafer/static/css/wafer.css

```diff
@@ -47,6 +47,10 @@
     border: 5px dashed #000;
 }
 
+.help-block {
+    color: #d9534f;
```

Comment:
That's quite red. Is that not going to be too close to the colour of errors?

Reply:
I picked it because it was the color of errors: `.help-block` is set on the validation error messages.

Reply:
Yeap, confirmed (and removed it from the patch).

Reply:
err it still seems to be here.